src/sentry/runner/commands/devservices.py

```diff
@@ -100,6 +101,19 @@ def up(project, exclude):
 
     if not sentry_options.get('symbolicator.enabled'):
         exclude |= {'symbolicator'}
+    elif sys.platform == 'darwin' and 'host.docker.internal' not in (sentry_options.get('system.internal-url-prefix') or ''):
```

Comment:
I think this is fine. Is `host.docker.internal` a hardcoded thing that comes from Docker for Mac? Or is it possible that others might change it?

It's also likely possible that we can just forcefully set this option for people on startup somewhere in here: https://github.com/getsentry/sentry/blob/master/src/sentry/runner/initializer.py

Especially if it's `darwin` only, it might be reasonable to just have:

```python
if sys.platform == 'darwin' and 'host.docker.internal' not in settings.SENTRY_OPTIONS['system.internal-url-prefix']:
    settings.SENTRY_OPTIONS['system.internal-url-prefix'] = 'http://host.docker.internal:%s' % settings.SENTRY_WEB_PORT
```

inside of the runner initializer and blindly assume these things are setup correctly.

Reply:
We can make this a dynamic default for `system.internal-url-prefix` but I would still like to run symbolicator without docker for dev purposes.

I'm trying to follow the trend here of making the user explicitly set the right settings instead of setting them automatically, at least that's what I saw when I set up Snuba with devservices.

Reply:
>I think this is fine. Is host.docker.internal a hardcoded thing that comes from Docker for Mac? Or is it possible that others might change it?

I don't know if it's possible to change, but I see no reason why people would change it. I'm also not sure whether "being on OS X" actually equates to "I run Docker for Mac", which is why it's a warning.

Reply:
We should also be firm on suggesting Docker for Mac. In 2019, there isn't really much other way to do it that is well supported. Unless you're doing something more legacy with docker-machine, but we can just say that's not supported.

Reply:
@mattrobenolt Ok, my personal preference is still to make users set the options they want to see, so one can look into the config and know the values there are actually the values that apply. I think it's impossible to miss this warning anyway.

Reply:
I think this should be addressed now after a private convo in slack